mycroft/skills/intent_service.py

```diff
@@ -296,14 +321,30 @@ def handle_utterance(self, message):
                     match = match_func(combined, lang, message)
                     if match:
                         break
+
             if match:
                 if match.skill_id:
-                    self.add_active_skill(match.skill_id)
-                    # If the service didn't report back the skill_id it
-                    # takes on the responsibility of making the skill "active"
+                    if self.skill_categories[match.skill_id] == 'undefined':
+                        """
+                        this has not been tested and not sure if this is 
+                        what we want. this will cause all old style skills 
+                        to send out a common play stop message. this is not
+                        current behavior. 
+                        """
+                        # stop last active common play skill
+                        msg_data = {"stop_type":"latest"}
+                        self.bus.emit(message.forward('play:stop', msg_data))
+
+                        self.add_active_skill(match.skill_id, 
+                                self.skill_categories[match.skill_id])
+                        LOG.error("IntentService:handle_utterance() - Stop Msg Sent, Adding old style skill (id=%s) to active_skills array=%s" % (match.skill_id, self.active_skills))
+                        # If the service didn't report back the skill_id it
+                        # takes on the responsibility of making the skill "active"
+                        # new style skills handle this, old style do not
 
                 # Launch skill if not handled by the match function
                 if match.intent_type:
+                    LOG.error("IntentService:handle_utterance() - Matched intent type! type:%s, \ndata:%s" % (match.intent_type, match.intent_data))
```

Comment:
Why is this an error?

Reply:
Its not - my mistake i left some extra logging in